.gitmodules

```diff
@@ -1,9 +1,12 @@
 [submodule "third-party/lammps"]
 	path = third-party/lammps
-	url = https://github.com/faasm/lammps.git
+    url = https://github.com/faasm/lammps.git
+    branch = faasm
 [submodule "third-party/kernels"]
 	path = third-party/kernels
 	url = https://github.com/faasm/Kernels.git
+    branch = Faasm
```

Comment:
Should this be in caps? AFAICT it's lowercase: https://github.com/faasm/Kernels/tree/faasm